recipes/entityx/all/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.15)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```

Could add a note why needs 3.15. Looking this recipe, 3.4 is more than enough.

---

It was not working correctly due to the runtime MT/MD, maybe after patching the build script it is not longer necessary?

---

3.1 is enough (upstream min version) if CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS is removed

---

Great, 3.1 so